File: 2559. Count Vowel Strings in Ranges/brute_force.ts
// Time Limit Exceeded
function vowelStrings(words: string[], queries: number[][]): number[] {
    const results = []
    for (const query of queries) {
        let counter = 0
        for (let i = query[0]; i <= query[1]; i++) {
            if (isVowel(words[i])) counter++
        }
        results.push(counter)
    }
    return results
};

function isVowel(word: string): boolean {
    const vowels = ['a', 'e', 'i', 'o', 'u']
    return vowels.includes(word.at(0)) && vowels.includes(word.at(-1))
}

File: 1930. Unique Length-3 Palindromic Subsequences/brute_force.py
# Time Limit Exceeded
def countPalindromicSubsequence(self, s: str) -> int:
    subsequences = set()
    for i in range(len(s)):
        for j in range(i + 1, len(s)):
            for k in range(j + 1, len(s)):
                if s[i] == s[k]:
                    subsequences.add(s[i] + s[j] + s[k])
    return len(subsequences)